readme.md

```diff
@@ -10,553 +19,44 @@ authentication and more to any API in minutes. Learn more at
-[![npm version](https://img.shields.io/npm/v/express-rate-limit.svg)](https://npmjs.org/package/express-rate-limit 'View this project on NPM')
-[![npm downloads](https://img.shields.io/npm/dm/express-rate-limit)](https://www.npmjs.com/package/express-rate-limit)
-
 Basic rate-limiting middleware for [Express](http://expressjs.com/). Use to
 limit repeated requests to public APIs and/or endpoints such as password reset.
 Plays nice with
 [express-slow-down](https://www.npmjs.com/package/express-slow-down).
 
-</div>
-
-## Use Cases
-
-Depending on your use case, you may want to switch to a different
-[store](#store).
-
-### Abuse Prevention
-
-The default `MemoryStore` is probably fine.
-
-### API Rate Limit Enforcement
-
-The default `MemoryStore` stores the hit counts for clients in memory, and is
-thus unsuitable for use when running multiple servers or processes.
-
-If you have multiple processes on a single server (via the
-[node:cluster](https://nodejs.org/api/cluster.html) module), the
-[`cluster-memory-store`](https://npmjs.com/package/@express-rate-limit/cluster-memory-store)
-will keep them all in sync without needing an external data store.
-
-If you have multiple servers, or want to maintain state across app restarts, use
-an external data store such as
-[redis](https://npmjs.com/package/rate-limit-redis),
-[memcached](https://npmjs.org/package/rate-limit-memcached), [etc.](#store).
-
-### Alternate Rate Limiters
-
-This module was designed to only handle the basics and didn't even support
-external stores initially. These other options all are excellent pieces of
-software and may be more appropriate for some situations:
-
-- [`rate-limiter-flexible`](https://www.npmjs.com/package/rate-limiter-flexible)
-- [`express-brute`](https://www.npmjs.com/package/express-brute)
-- [`rate-limiter`](https://www.npmjs.com/package/express-limiter)
-
-## Installation
-
-From the npm registry:
-
-```sh
-# Using npm
-> npm install express-rate-limit
-# Using yarn or pnpm
-> yarn/pnpm add express-rate-limit
-```
-
-From Github Releases:
-
-```sh
-# Using npm
-> npm install https://github.com/express-rate-limit/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
-# Using yarn or pnpm
-> yarn/pnpm add https://github.com/express-rate-limit/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
-```
-
-Replace `{version}` with the version of the package that you want to your, e.g.:
-`6.0.0`.
-
 ## Usage
 
-### Importing
-
-This library is provided in ESM as well as CJS forms, and works with both
-Javascript and Typescript projects.
-
-**This package requires you to use Node 16 or above.**
-
-Import it in a CommonJS project (`type: commonjs` or no `type` field in
-`package.json`) as follows:
+An example of its use is as follows:
 
 ```ts
-const { rateLimit } = require('express-rate-limit')
-```
+// Run `npm install express-rate-limit` to install the library.
```

Comment:
I think we can omit this from the readme. Most folks who've gotten to the point of needing a rate-limiter already know how to do this, and we have it in the full docs for the ones who don't.